addons/mail/static/src/models/partner/partner.js

```diff
@@ -190,7 +190,8 @@ function factory(dependencies) {
                     continue;
                 }
                 if (
-                    (partner.nameOrDisplayName && cleanSearchTerm(partner.nameOrDisplayName).includes(cleanedSearchTerm)) ||
+                    partner.name &&
```

Comment:
I meant to move it on the other condition of the method, or maybe even make a separate condition for it, I don't like having a mix of and/or here.

Reply:
alright